lib/Github/Api/Migrations/SourceImport.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Github\Api\Migrations;
+
+use Github\Api\AbstractApi;
+use Github\Api\AcceptHeaderTrait;
+use Github\Exception\InvalidArgumentException;
+use Github\Exception\MissingArgumentException;
+
+/**
+ * Support for Migrations.
+ *
+ * @link   http://developer.github.com/v3/migrations
+ *
+ * @author Sebastian Berm <sebastian@sebsoft.nl>
+ */
+class SourceImport extends AbstractApi
+{
+    use AcceptHeaderTrait;
+
+    /**
+     * Start import of repo from other VCS.
+     *
+     * @link https://developer.github.com/v3/migrations/source_imports/
+     *
+     * return array
+     */
+    public function start($params, $owner, $repoName)
```

Comment:
I would move the `$params` parameter to the end of the function parameter list, to be in line with the rest of the code. This parameter can also be typehinted as array!